docs/human-interface-guidelines.md

```diff
@@ -266,64 +266,64 @@ For more info, see the FreeDesktop.Org [menu entry](http://standards.freedesktop
 * If the item acted upon is the only item currently visible, such as a web page or a contact in Dexter, contractor can be displayed in a toolbar menu.
 * Never display the target app's name. The menu string is the intended display name for that app. Doing so can lead to redundant messages such as, "Set image as wallpaper (wallpaper)" and it is always irrelevant to a user's needs.
 
 ## Dock Integration {#dock-integration}
 
-By integration with Pantheon's dock, apps are able to communicate their status to the user at a glance.
+Apps integrated with Pantheon's dock communicate their status to the user at a glance.
 
 ![](/images/docs/human-interface-guidelines/dock-integration/dock.png)
 
 ### Progressbars {#progressbars}
 
-A progressbar must be unambiguous in its use, referring to a single specific task. For example, a progressbar can be used to indicate the status of a file transfer or of a lengthy process like encoding. A progressbar should not be used to compound the progress of multiple types of actions.
+Make progress bars unambiguous by referring to a single, specific task. For example, use progress bars to indicate the status of lengthy processes like file transfers and encoding. Do not use progress bars to compound the progress of different types of action.
 
-* **Good Example**: Installation progress in Software Center
+* **Good Example**: Installation progress in AppCenter
 * **Bad Example**: Combined progress of downloading an album, burning a CD, and syncing a mobile device in Noise
 
 ### Badges {#badges}
 
-A badge shows a count of actionable items which your app manages. It's purpose is to inform the user that there are items that require user attention or action without being obtrusive. This is a passive notification. A badge should not show totals or rarely changing counters. If the badge is not easily dismissed when the user views your app, it is likely that this is not a good use of a badge.
+A badge shows a count of actionable items managed by your app. Its purpose is to inform the user that there are items that require user attention or action without being obtrusive. This is a passive notification. A badge should not show totals or rarely changing counters. If the badge is not easily dismissed when the user views your app, it is likely that this is not a good use of a badge.
 
-* **Good Example**: Unread messages in Geary
+* **Good Example**: Unread messages in Mail
 * **Bad Example**: Total number of Photos in Shotwell
 
 ## System Indicators {#system-indicators}
 
-Indicators are small icons that live on the top panel. They give users a place to glance for a quick indication of various settings or events. Clicking the icon shows a small menu with related actions available to the user.
+Indicators are small icons that live in the menu bar (wingpanel). They give users a place to glance for a quick indication of various settings or events. Clicking the icon shows a small menu with related actions available to the user.
```

Comment:
We use the term "panel" and this dates back to the GNOME 2 days. We don't say menu bar to refer to the panel. We use menubar to refer to the Gtk+ widget `Gtk.Menubar` which we recommend against using in applications.